app/lib/esClient.js

```diff
@@ -33,11 +50,28 @@ async function getGp(id) {
   }
 }
 
+function getHitsResult(result, code) {
+  const hits = result.hits && result.hits.hits;
+  if (hits.length > 1) {
+    log.warning({ hits }, `Warning multiple records returned for ${code}, expecting only one`);
```

Comment:
I assume this is ok as a warning for information purposes rather than a proper error? How many times does this happen? And how do you know which one to choose (I can see the first one is selected below).

Reply:
Currently there are 40 GPs with duplicate ODS codes in Syndication. We're still deciding on how to best handle these, the data team has been notified so they can be fixed.